backend/models/volcon_db.py

```diff
@@ -57,6 +57,18 @@ class Organization(User):
         'polymorphic_identity': 'organization'
     }
 
+    def to_dict(self):
+        """Function to Convert entry data to a python dict"""
+        return {
+            'org_id': self.org_id,
+            'org_name': self.org_name,
+            'location': self.location,
+            'biography': self.biography,
+            'profile_logo': self.profile_logo,
```

Comment:
Remove this (profile_logo) from organization table and add it into the user table because both roles have a profile picture. organization uses it as a logo and volunteer uses it as a profilePicture